secboot/secboot_tpm.go

```diff
@@ -21,27 +21,42 @@
 package secboot
 
 import (
+	"crypto/rand"
 	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 
+	"github.com/canonical/go-tpm2"
 	sb "github.com/snapcore/secboot"
 	"golang.org/x/xerrors"
 
 	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/boot"
 	"github.com/snapcore/snapd/bootloader/efi"
+	// TODO: UC20: move/merge partition with gadget
+	"github.com/snapcore/snapd/cmd/snap-bootstrap/partition"
 	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/randutil"
 )
 
+const (
+	// Handles are in the block reserved for owner objects (0x01800000 - 0x01bfffff)
+	pinHandle = 0x01880000
```

Comment:
Not that it matters, and it doesn't cause any backwards compatibility issues, but this value seems to have changed (it was 0x01800000 previously)

Reply:
You have very sharp eyes. Yes, this was arbitrarily changed for no reason except not being at the very start of the reserved area.